oauth2/build.gradle

```diff
@@ -31,6 +31,7 @@ android {
         }
     }
     compileOptions {
+        coreLibraryDesugaringEnabled true
```

Comment:
After gradle update, I started getting compile errors unless I added desugaring to oauth2 and web-authentication-ui modules